src/Core/Services/Implementations/LocalSendStorageService.cs

```diff
@@ -21,17 +26,21 @@ public class LocalSendStorageService : ISendFileStorageService
         public async Task UploadNewFileAsync(Stream stream, Send send, string fileId)
         {
             await InitAsync();
-            using (var fs = File.Create($"{_baseDirPath}/{fileId}"))
+            var path = FilePath(send, fileId);
+            Directory.CreateDirectory(Path.GetDirectoryName(path));
```

Comment:
Any reason you're changing this to create a directory per Send, when a Send would only ever have a single file?

Reply:
I changed this just to match AzureSendFileStorageService's structure. We could absolutely store this at the base directory with no issues.

This also allows eventually supporting multiple files for a send if we want.

Reply:
That's fine, but in that case when deleting a file, before deleting a directory should check if it's empty first.